contrib/kube-prometheus/jsonnetfile.lock.json

```diff
@@ -8,7 +8,7 @@
                     "subdir": "contrib/kube-prometheus/jsonnet/kube-prometheus"
                 }
             },
-            "version": "df002d09f7b7a50321786c4f19c70d371494410b"
+            "version": "4a9bd8093cae26b8ad958c707f0a5d141f45e648"
```

Comment:
You don't need to update this dependency for changing the kops part. If you remove this change the CI will work and we can merge.

Reply:
Sorry, dropped the commit.